src/util/test/String/LevenshteinTest.js

```diff
@@ -1,7 +1,7 @@
 require('../../lib/String/levenshtein');
 const expect = require('chai').expect;
 
-describe('Levenshtein distance', function () {
+describe('Levenshtein distance', () => {
```

Comment:
Please do not use arrow functions in the `describe` function call. The reason is explained here: https://mochajs.org/#arrow-functions